### DISCUSSION TITLE
Sending a plain text response?

### DISCUSSION THREAD

**Discussion by vchav73** [zauberzeug/nicegui#3937] [Q&A] [answered]
### Question

I need to add a URL path to my existing NiceGUI app which returns a plain text document (text/plain MIME type) to the browser. It's used to automatically verify correct deployment for a CI / CD pipeline. Is there a way to do this via NiceGUI? I've searched the documentation but haven't found such a function. Thank you.

**Comment by rodja** [comment, marked as answer]
NiceGUI is build upon FastAPI. So you can simply add RESTful endpoints to the app:

```py
from nicegui import ui, app

ui.label('Hello')

@app.get('/status')
def status():
    return {'status': 'ok'}

ui.run()

**Reply by vchav73** [reply]
That worked. Thank you!
